src/output-json-alert.c

```diff
@@ -184,6 +186,35 @@ static void AlertJsonDnp3(const Flow *f, json_t *js)
     return;
 }
 
+static void AlertJsonDns(const Flow *f, json_t *js)
+{
+#ifndef HAVE_RUST
```

Comment:
would be nice to have this for rust too. Not a requirement for this PR, but maybe something to try and get into 4.1?